packages/playground/assets/js/playground.js

```diff
@@ -53,9 +50,29 @@
 
 	await client.isReady();
 
+	// Login as the current user without a password
+	await client.writeFile(
+		'/wordpress/playground-login.php',
+		`<?php
+		require_once( dirname( __FILE__ ) . '/wp-load.php' );
+		if ( is_user_logged_in() ) {
+			return;
+		}
+		$user = get_user_by( 'id', ${playground.userId} );
+		if( $user ) {
+			wp_set_current_user( $user->ID, $user->user_login );
+			wp_set_auth_cookie( $user->ID );
+			do_action( 'wp_login', $user->user_login, $user );
+		}`
+	);
+	await client.request({
+		url: '/playground-login.php',
+	});
+	await client.unlink('/wordpress/playground-login.php');
+
```

Comment:
@adamziel turns out that we need to run the login code using a request to correctly set the cookie.

Reply:
This code would also work in the Playground login step if we ever want to support passwordless logins.

Reply:
https://github.com/WordPress/blueprints-library/issues/91